ui-v2/app/styles/components/table.scss

```diff
@@ -18,8 +19,8 @@ td dt.critical {
 table:not(.sessions) tr {
   cursor: pointer;
 }
-th,
-td {
+thead tr,
```

Comment:
This is probably just me showing my age 😄 , but how sure are you that this is fine in IE?

https://stackoverflow.com/questions/2832260/why-border-of-tr-not-showing-in-ie

I know the link is for IE7 (I know I know 😆 ), but I'm not sure off the top of my head at in what version they fixed it, if it was me I'd like to check. I'm sure its fine I would imagine its fixed by now, but just worth pointing out. Plus I suppose there is not much benefit to doing this apart from it reading better and being more 'correct' semantically, I think it also adds a few bytes to the download.

Reply:
I am not sure how IE handles this, but I can look into it. What versions of IE do we actively support?

>Plus I suppose there is not much benefit to doing this apart from it reading better

I changed this because I noticed that the items in the table were not vertically aligned in the center of the table row. When I adjusted that alignment (using a couple different choices), the `td`s moved separately, not as a row unit. Since the border-bottom is attached to the `td` here in this line, the row ended up getting buggy:

<img width="150" alt="screen shot 2018-07-20 at 10 14 50 am" src="https://user-images.githubusercontent.com/6323933/43010362-d77258aa-8c05-11e8-9612-d8875e644409.png">

I decided to eventually not try to solve the alignment issue in this PR, which is supposed to be focused on grays (I like to make sure my PRs solve one issue at a time), but I kept this change because it seemed like the right thing to do. If it's a big deal, we can definitely revert and come back to this later!